A3A/addons/ultimate/config.cpp

```diff
@@ -14,23 +15,38 @@ class CfgPatches
         authorUrl = "";
         VERSION_CONFIG;
     };
+    class D37_cram
+    {
+        units[] = {"CRAM_Fake_PlaneTGT"};
+		weapons[] = {};
+		requiredVersion = 0.1;
+		requiredAddons[] = {"A3_Static_F_Jets_AAA_System_01"};
+    };
+    class D37_dome
+    {
+        units[] = {"B_SAM_System_01_F_DOME"};
+		weapons[] = {"B_SAM_System_01_F_DOME"};
+		requiredVersion = 0.1;
+		requiredAddons[] = {"A3_Static_F_Jets_SAM_System_01","A3_Static_F_Jets_SAM_System_02"};
+    };
```

Comment:
You'll need to add CfgVehicles entries to the units[] array if you want them to be spawnable in zeus

Reply:
We kinda don't want to. I mean, every turret added into config, is auto C-RAM/IRON-DOME capable, so if you want, you can spawn reg. ones and disable functionality manually via interaction